app/frontend/src/utils/stripMarkdown.ts

```diff
@@ -0,0 +1,85 @@
+/* eslint-disable no-useless-escape */
```

Comment:
Was getting this error but too scared to remove them. I think it's not correct?

Reply:
Which bit is the linting erroring on?

Reply:
Also how come this source code is slightly different from the repo you linked? I assume you didn't just install it since it comes with another markdown editor that we don't need?

Reply:
Whoops! It's actually https://github.com/stiang/remove-markdown

Reply:
-1 to the line number
```
/home/lucas/couchers/app/frontend/src/utils/stripMarkdown.ts
  33:24  warning  Unnecessary escape character: \*  no-useless-escape
  33:28  warning  Unnecessary escape character: \+  no-useless-escape
  36:49  warning  Unnecessary escape character: \*  no-useless-escape
  36:53  warning  Unnecessary escape character: \+  no-useless-escape
  53:20  warning  Unnecessary escape character: \-  no-useless-escape
  55:27  warning  Unnecessary escape character: \:  no-useless-escape
  58:17  warning  Unnecessary escape character: \!  no-useless-escape
  58:29  warning  Unnecessary escape character: \[  no-useless-escape
  58:31  warning  Unnecessary escape character: \(  no-useless-escape
  58:40  warning  Unnecessary escape character: \)  no-useless-escape
  60:27  warning  Unnecessary escape character: \[  no-useless-escape
  60:29  warning  Unnecessary escape character: \(  no-useless-escape
  60:38  warning  Unnecessary escape character: \)  no-useless-escape
  71:19  warning  Unnecessary escape character: \*  no-useless-escape
  72:19  warning  Unnecessary escape character: \*  no-useless-escape
```

Reply:
> In most regex flavors, the only special characters or metacharacters inside a character class are the closing bracket ], the backslash \, the caret ^, and the hyphen -. The usual metacharacters are normal characters inside a character class, and do not need to be escaped by a backslash. To search for a star or plus, use [+*]. Your regex will work fine if you escape the regular metacharacters inside a character class, but doing so significantly reduces readability.

I will fix them.